tests/gtest/avif_fuzztest_enc_dec.cc

```diff
@@ -49,12 +49,12 @@ void EncodeDecodeValid(ImagePtr image, EncoderPtr encoder, DecoderPtr decoder) {
   const avifResult encoder_result =
       avifEncoderWrite(encoder.get(), image.get(), &encoded_data);
   ASSERT_EQ(encoder_result, AVIF_RESULT_OK)
-      << avifResultToString(encoder_result);
+      << avifResultToString(encoder_result) << " " << encoder->diag.error;
```

Comment:
Nit: change `" "` to `": "` so that the output looks like `ERROR_CODE: error message`.

Also in line 57.